### PR TITLE
Update WSUS example

### PR DESCRIPTION
**Pull request by magrande**
Update WSUS example by fixing the PS script, json config file and by adding a README linking to the supported article.

### CODE REVIEW

File: wsus/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Set up WSUS by using the automation script
+
+This content supports the [Plan deployment for updating Windows VMs in Azure](https://github.com/MicrosoftDocs/architecture-center/tree/master/docs/example-scenario/wsus) Azure Architecture Center article. For information about these files please see that article.
```
Comment (ckittel) [inline wsus/README.md:3]: Can you please link to the live docs page instead of the the github repo's version of this page?

**Review by ckittel — CHANGES_REQUESTED** [review]
Other than that, I guess if this is all that needed to be refreshed, then it looks fine.

File: wsus/README.md (hunk 1 above)
Comment (magrande) [reply wsus/README.md]: Done

**Review by ckittel — APPROVED** [review]
🚀

**Pull request merged**